hknweb/candidate/models.py

```diff
@@ -56,6 +56,15 @@ def rejected(self):
     def __str__(self):
         return self.name
 
+class CandidateForm(models.Model):
+
+    name           = models.CharField(max_length=MAX_STRLEN, default='')
+    duedate        = models.DateTimeField(default=timezone.now)
+    link           = models.CharField(max_length=MAX_STRLEN, default='')
```

Comment:
Maybe make this a text field instead? In case of really long links? I'm not sure on this one